### ISSUE TITLE
Improving performance

### ISSUE THREAD

**Issue by Arjunsankarlal** [galsang/BIMPM-pytorch#8]
I have trained a model and I am able to get way better results than the previous method that I was using. But now I face performance issues. For some queries, the model takes more than 6 seconds to do prediction. Any heads up to tuning the prediction time would be helpful! :)

**Comment by Albert-Ma** [comment]
What's the param you used for QQP and SNLI

**Comment by Arjunsankarlal** [comment]
The default params itself works well, I reduced the batch size to 32, since 64 was causing memory issues while training. Well, instead of using Glove embeddings, try using FastText Crawl embeddings, they give 0.5% increase in accuracy.
